Add capability for new devices

justinkumpe opened the pull request:
Set hasHapticFeedback to use CoreHaptics to detect if device has haptic engine if device is using iOS 13+ instead of having to rely on a device list that has to be updated each time new iPhones are released.

justinkumpe committed:
Merge pull request #1 from iSapozhnik/master

Merge pull request #15 from justinkumpe/master

justinkumpe committed:
adding capability to use CoreHaptics to detect if capable of haptics for iOS 13+ instead of having to rely on a list of devices

justinkumpe commented:
Also bumped version number in podsec file. If merged please push new version to cocoapods.

justinkumpe commented:
@iSapozhnik can you review this when you get a chance please?

iSapozhnik commented:
Yes will do today or tomorrow. Thanks for the reminder

iSapozhnik commented on Package.swift:
```diff
@@ -4,12 +4,12 @@ import PackageDescription
     name: "Haptico",
     platforms: [
-        .iOS(.v9),
+        .iOS(.v13),
```
Won't it make it impossible to use Haptico on iOS prior to iOS 13?

justinkumpe replied:
Glad you asked. I had changed that to require iOS 13 then went back and fixed it where it could still be compatible with 9+ and forgot to change the required version back. I pushed a new commit to update that back to v9. The changes I made to support the newer devices would require iOS 13+ but those devices were shipped with OS higher than 13 and I did wrap the changes in if #available(iOS 13.0, *) so it should be fine.

iSapozhnik commented on Example/Podfile:
```diff
@@ -1,5 +1,5 @@
 use_frameworks!
-platform :ios, '9.0'
+platform :ios, '13.0'
```
With this change, we will drop support of iOS versions prior to iOS 13, right?

justinkumpe replied:
Glad you asked. I had changed that to require iOS 13 then went back and fixed it where it could still be compatible with 9+ and forgot to change the required version back. I pushed a new commit to update that back to 9.0. The changes I made to support the newer devices would require iOS 13+ but those devices were shipped with OS higher than 13 and I did wrap the changes in if #available(iOS 13.0, *) so it should be fine.

justinkumpe committed:
Update Podfile

justinkumpe committed:
Update Package.swift

justinkumpe committed:
Update Haptico.podspec

justinkumpe commented:
@iSapozhnik Requested changes have been made. It was actually a mistake on my part, not sure why I changed the iOS compatibility to 13 when I already accounted for it in the code so I changed it back to iOS 9.

iSapozhnik commented:
@justinkumpe Thanks for the contribution. Unfortunately, I can not push new podspec - Heroku, the underlying service used by Cocoapods, is having some issues... Can't create a session. 
![image](https://user-images.githubusercontent.com/1089384/201521983-0299aaa6-e529-438f-96c2-786b9fa40bcb.png)

Will try again on Monday.

iSapozhnik commented:
@justinkumpe Finally! Thanks for your patience!

```
 🎉  Congrats

 🚀  Haptico (1.2.0) successfully published
 📅  November 17th, 00:30
 🌎  https://cocoapods.org/pods/Haptico
 👍  Tell your friends!
```

justinkumpe commented:
Cool, thanks
